cdrs-tokio/src/cluster/token_factory.rs

```diff
@@ -2,7 +2,7 @@ use bytes::Buf;
 use std::cmp::min;
 use std::num::Wrapping;
 
-use crate::cluster::Murmur3Token;
+use cassandra_protocol::query::query_params::Murmur3Token;
 
 const C1: Wrapping<i64> = Wrapping(0x87c3_7b91_1142_53d5_u64 as i64);
 const C2: Wrapping<i64> = Wrapping(0x4cf5_ad43_2745_937f_u64 as i64);
```

Comment:
The `generate_murmur3_token` function should be moved to `cassandra_protocol` too

Reply:
token_factory isnt exposed in the cdrs-tokio API.
If we do want to expose it we should do that in a follow up PR.